src/Microsoft.TestPlatform.PlatformAbstractions/Interfaces/System/IAssemblyResolver.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The AssemblyResolver interface.
+    /// </summary>
+    public interface IAssemblyResolver : IDisposable
```

Comment:
Assembly resolution is probably a runtime aspect (and not a system wide aspect). Suggest to move to Interfaces/Runtime.